OWNERS_ALIASES

```diff
@@ -28,14 +26,17 @@ aliases:
   - maximilien
   - navidshaikh
   - rhuss
-  conformance-wg-leads:
-  - tayarani
+  conformance-task-force-leads:
+  - omerbensaadon
+  - salaboy
   conformance-writers:
-  - tayarani
+  - omerbensaadon
+  - salaboy
   docs-reviewers:
   - RichardJJG
   - jonatasbaldin
   - mpetason
+  - nainaz
```

Comment:
@pmorie and this was introduced [here](https://github.com/knative/community/pull/765)